src/OrchardCore.Modules/OrchardCore.Https/OrchardCore.Https.csproj

```diff
@@ -5,6 +5,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.HttpsPolicy" Version="2.1.0" />
```

Comment:
I _think_ the version should be specified in https://github.com/OrchardCMS/OrchardCore/blob/dev/src/OrchardCore.Build/Dependencies.AspNetCore.props not here.